osd_architecture/osd_architecture_sub/osd-architecture-models-gcp.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="osd-architecture-models-gcp"]
+= GCP architecture models
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: osd-architecture-models-gcp
+
+toc::[]
+
+{product-title} on {GCP} has two installation offerings. The architecture supports the following network configuration types:
+
+* Public network
+* Private network
+* GCP Private Service Connect
+
+.OSD deployed on public and private networks
+image::osd-gcp-architecture.png[OSD Architecture]
```

Comment:
I assume the right set of nodes on the right in each of the pictures should be infra nodes, similar to the [ROSA version of the picture](https://docs.openshift.com/rosa/rosa_architecture/rosa_architecture_sub/rosa-architecture-models.html#rosa-classic-architecture-networks_rosa-architecture-models).